portfolio/src/main/java/com/google/sps/Comment.java

```diff
@@ -0,0 +1,5 @@
+package com.google.sps;
+public class Comment{
+    public String commentor;
+    public String comment;
+}
```

Comment:
Please add getters and setters. It's good practice to use private variables and user getter/setter method to retrieve/populate these variables. 
This is a good artice - https://dzone.com/articles/why-should-i-write-getters-and-setters